lib/howl/application.moon

```diff
@@ -4,13 +4,20 @@
 ffi = require 'ffi'
 
 import Window, Editor, theme from howl.ui
-import Buffer, Settings, mode, bundle, bindings, keymap, signal, interact, timer, clipboard from howl
+import Buffer, Settings, mode, bundle, bindings, keymap, signal, interact, timer, clipboard, config from howl
 import File, Process from howl.io
 import PropertyObject from howl.aux.moon
 Gtk = require 'ljglibs.gtk'
 
 append = table.insert
 
+config.define
```

Comment:
As a style nit-picking I'd prefer if this was moved down at the end of the file instead, together with the signal definitions.